### ISSUE TITLE
No Notification for Gloves of Silence Breaking

### ISSUE THREAD

**Issue by lmack22** [RDieleman/runelite-pickpocket-helper#3]
It would be nice if there was an optional notification when your gloves of silence break, similar to the dodgy necklace notification.

Great plugin, thanks so much for making this!

**Comment by RDieleman** [comment]
@lmack22 I'm glad you enjoy the plugin! I'll take a look at adding a gloves of silence notification.

**Comment by RDieleman** [comment]
@lmack22 Should be working once Runelite adds the update. I can't test it because of the Ardy hard diaries on my account, so let me know if it doesn't work.

**Comment by lmack22** [comment]
I'm not seeing it yet, but I'll let you know. Thanks!

**Comment by lmack22** [comment]
I'm not very familiar with RuneLite plugin updates, but I'm assuming it should be showing by now right?
![Screenshot 2023-01-24 224859](https://user-images.githubusercontent.com/122746560/214482393-7122a481-c86f-4ea4-83ac-7ebf3955a384.jpg)

**Comment by RDieleman** [comment]
@lmack22 Yeah, that's where the alert will be added. However, before they are available in-game these changes need Runelite's approval, which can take a little while, unfortunately. You can see the status of that [here](https://github.com/runelite/plugin-hub/pull/3796).

Otherwise, you can clone this repository and run it in a development environment which would allow you to use it right now.

**Comment by lmack22** [comment]
Sorry for the late reply on this. I finally got around to testing it today, and it wasn't working for me.
